ord_schema/interface/query.py

```diff
@@ -20,15 +47,252 @@
 from absl import logging
 import psycopg2
 from psycopg2 import sql
+from rdkit import Chem
 
 from ord_schema.proto import dataset_pb2
 from ord_schema.proto import reaction_pb2
 
-# pylint: disable=too-many-locals
 
+def fetch_results(cursor):
+    """Fetches query results.
+
+    Args:
+        cursor: psycopg.cursor instance.
+
+    Returns:
+        Dict mapping reaction IDs to serialized Reaction protos.
+    """
+    reactions = {}
+    for reaction_id, serialized in cursor:
+        reactions[reaction_id] = serialized
```

Comment:
Is result ranking never meaningful? We'll never care about ORDER BY?

Reply:
I don't think the order is ever meaningful.